Makefile

```diff
@@ -226,18 +232,39 @@ release-image-tag: ## Prints the image tag of the release image
 ## End-to-end (E2E) Testing ##
 ##############################
 
-e2e-setup: ## Setup end-to-end tests
+e2e-build: ## Build the e2e Docker image, if not already built, using ./e2e/Dockerfile
+	docker build -t playwright-e2e -f ./e2e/Dockerfile .
+
+e2e-copy-report: ## Copy the Playwright report from the container to local
+	docker cp playwright-e2e-container:/e2e/playwright-report ./e2e/playwright-report
+
+e2e-clean-report: ## Remove the local ./e2e/playwright-report folder and its contents
+	rm -rf ./e2e/playwright-report
+
+e2e-delete-image: ## Delete the Docker image for e2e tests
+	@docker rmi -f playwright-e2e 2>/dev/null || echo "Docker image playwright-e2e does not exist, skipping."
+
+e2e-run: ## Run the Playwright tests in a Docker container and copy the report locally
+e2e-run: e2e-build
```

Comment:
I left this comment in my previous review. It's confusing to have `e2e-run` and `e2e-test`. What's the difference? Why does `run` mean run in Docker and `test` mean run natively? Please rename to `e2e-test` and `e2e-test-native`